portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.servlets;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.gson.Gson;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+class CommentsServletTest {
+  private CommentsServlet servlet;
+  private MockHttpServletRequest request;
+  private MockHttpServletResponse response;
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  @BeforeEach
+  public void setUp() {
+    servlet = new CommentsServlet();
+    request = new MockHttpServletRequest();
+    response = new MockHttpServletResponse();
+    helper.setUp();
+  }
+
+  @Test
+  void testPost() {
+    Gson gson = new Gson();
+    Comment comment = new Comment();
+    request.setMethod("POST");
+    request.setContentType("text/html");
+    request.addParameter("username", "paul");
+    request.addParameter("text", "A nice comment");
+    try {
+      servlet.doPost(request, response);
+      assertEquals(response.getStatus(), 200);
```

Comment:
Are there cases where the server throws an error? If yes, then we should be testing them too.

Reply:
There are cases when server throws exception, yes. Do you mean I have to check that the correct exception was thrown?

Reply:
Nah, I was talking about the response status returned. You also have to check for failure cases in a test.

Reply:
Sure, this PR is some basic test setup, in branches `writing-server-tests` and `more-server-tests` I do more thorough testing. Here is setup only and an example of how Im going to write tests.

Reply:
Ah, I see. okay.